javascript/javascript3/week3/shopping-cart/main.js

```diff
@@ -0,0 +1,118 @@
+class Product {
+  constructor(name, price) {
+    this.name = name;
+    this.price = price;
+  }
+
+  convertToCurrency(currency) {
+    fetch(`https://api.exchangerate-api.com/v4/latest/DKK`)
+      .then((response) => response.json())
+      .then((data) => {
+        let currencyConvert = (data.rates[currency] * this.price).toFixed(2);
+        console.log(
+          `Price of ${this.name} in ${currency} is ${currencyConvert}`
+        );
+      });
+  }
+}
+
+class ShoppingCart {
+  constructor() {
+    this.products = [];
+  }
+
+  addProduct(product) {
+    // Implement functionality here
+    this.products.push(product);
+  }
+
+  removeProduct(product) {
+    // Implement functionality here
+    const removedProduct = this.products.filter(
+      (itemname) => product !== itemname
+    );
+    this.products = removedProduct;
+  }
+
+  searchProduct(productName) {
+    // Implement functionality here
+    let SearchProductName = this.products.filter(
+      (itemname) => itemname === productName
+    );
+    console.log(SearchProductName);
+  }
+
+  getTotal() {
+    // Implement functionality here
+    const pricelist = this.products.map((product) => product.price);
```

Comment:
good variable name, but it is not necessary to put parentheses around `(product)`